### ISSUE TITLE
'init' command doesn't work

### ISSUE THREAD

**Issue by cttillman** [trustcrypto/python-onlykey#63]
```


Press the Enter key once you are done



Press the Enter key once you are done


 for second profile

Press the Enter key once you are done
 for second profile


Press the Enter key once you are done




Press the Enter key once you are done



Press the Enter key once you are done


```

These prompts are vague and unclear. What exactly am I supposed to do?

I do not know if it is relevant to the main problem, but I also cannot list credentials without getting an error saying ```AttributeError: 'Fido2Client' object has no attribute 'client_pin'```

Being able to store my private keys within the token sounds like an awesome idea if this CLI worked the way it is supposed to. I am running it on the latest version of Debian.

**Comment by onlykey** [comment]
Have you followed setup instruction here for OnlyKey https://onlykey.io/start or here for OnlyKey DUO https://onlykey.io/duo

**Comment by cttillman** [comment]
Set-pin gives the same error.

> Set new FIDO PIN, this is the PIN entered via keyboard and used for FIDO2 register/login (not the OnlyKey PIN entered on device).

The only PIN I created during setup was the device PIN. Even if it is already set, why can't I make a new one with this command?

**Comment by Kleptoid** [comment]
I'm getting the same error attempting to list credentials or set the fido2 PIN using onlykey-cli on NixOS. The onlykey is already initialised, it all works just fine on Ubuntu.

**Comment by onlykey** [comment]
@Kleptoid You can only list credentials if there are FIDO2 resident credentials to list and a FIDO2 PIN has been set.

**Comment by Kleptoid** [comment]
> @Kleptoid You can only list credentials if there are FIDO2 resident credentials to list and a FIDO2 PIN has been set.

Yes, there are credentials and a pin has been set on the Ubuntu machine, on that machine listing credentials works as expected. On the NixOS machine it doesn't, giving the stated error. It may be a problem with the NixOS package or environment, but since it seems to be happening on someone's Debian as well I though I'd mention it :)
